.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,5 +11,11 @@
 * Enter your extended release note here
 ```
 
+If you want this PR to close an issue add the following:
+"fixes #<issue nubmber>"
+either in this pull request message or the commit message.
```

Comment:
Prefer not to encourage embedding PR #s in commit messages.  Can you reword as "If you want this PR to automatically close an issue when it is merged, add the following to the pull request message:".  Maybe also note that a PR can close multiple issues.